diff --git a/components/fatfs/diskio/diskio.c b/components/fatfs/diskio/diskio.c
index c807516..49a4c76 100644
--- a/components/fatfs/diskio/diskio.c
+++ b/components/fatfs/diskio/diskio.c
@@ -20,8 +20,8 @@ static ff_diskio_impl_t * s_impls[FF_VOLUMES] = { NULL };
 
 #if FF_MULTI_PARTITION		/* Multiple partition configuration */
 PARTITION VolToPart[] = {
-    {0, 0},    /* Logical drive 0 ==> Physical drive 0, auto detection */
-    {1, 0}     /* Logical drive 1 ==> Physical drive 1, auto detection */
+    {0, 1},    /* Logical drive 0 ==> Physical drive 0, force */
+    {1, 2}     /* Logical drive 1 ==> Physical drive 1, force */
 };
 #endif
 
